### PR TITLE
Edited contribution language in weekly template

### PR DESCRIPTION
**Pull request by RichardLitt**
How does this sound?

### CODE REVIEW

File: template.md [hunk 1]
```diff
@@ -22,7 +22,9 @@ Across the entire IPFS GitHub organization, the following people have committed
 
 - __Contributors__
 
-Thanks, and see you next week!  If you have cool things to share for the next weekly, drop us a line in the [next weekly sprint issue](//github.com/ipfs/pm/issues/83)!
+This newsletter is also made by contributors. If you have cool things to share for the next weekly, drop a comment about it in the [next weekly sprint issue](//github.com/ipfs/pm/issues/83)! The more people mention items they want to see in the weekly there, the easier it is to make this and send it out.
```
Comment (hackergrrl) [inline template.md]: Is the _next weekly sprint issue_ link correct?

Comment (dignifiedquire) [inline template.md]: - maybe better: "made by **the** contributors" ? 
- A bit confused on what you are trying to say

Comment (RichardLitt) [inline template.md]: How does that look?

**Comment by RichardLitt** [comment]
@dignifiedquire Clarified that it is a community effort. Should be good now.

**Comment by dignifiedquire** [comment]
@RichardLitt LGTM

**Pull request merged**